PySDM/products/housekeeping/super_droplet_count_per_gridbox.py

```diff
@@ -1,17 +1,31 @@
 """
 super-droplet count per gridbox (dimensionless)
 """
+import numba
+
+from PySDM.backends.impl_numba.conf import JIT_FLAGS
 from PySDM.products.impl.product import Product
 
 
 class SuperDropletCountPerGridbox(Product):
     def __init__(self, unit="dimensionless", name=None):
         super().__init__(unit=unit, name=name)
+        self.impl = None
+
+    def register(self, builder):
+        super().register(builder)
+
+        @numba.njit(**{**JIT_FLAGS, "fastmath": builder.formulae.fastmath})
+        def impl(cell_start, buffer_ravel):
+            n_cell = cell_start.shape[0] - 1
+            for i in numba.prange(n_cell):  # pylint: disable=not-an-iterable
+                buffer_ravel[i] = cell_start[i + 1] - cell_start[i]
+
+        self.impl = impl
```

Comment:
It seems that it should look like this:
```suggestion
        self._impl = impl
```

https://github.com/open-atmos/PySDM/blob/a1f4b54b348d5626080fa146fbb92e5be869c2ef/PySDM/products/impl/product.py#L101

Reply:
`_impl` and `impl` are completely different routines! Which indeed calls for a name change...

Reply:
done